packages/Pagination/src/components/ArrowItem/ArrowItem.js

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+import PropTypes from "prop-types";
+import useI18n from "@paprika/l10n/lib/useI18n";
+import { getArrowIcon } from "./ArrowItem.styles";
+import * as sc from "./ArrowItem.styles";
+
+const propTypes = {
+  /* Description of the ArrowItem for assistive technology */
+  a11yText: PropTypes.string,
+
+  /** To set the ArrowItem as disabled */
+  isDisabled: PropTypes.bool,
+
+  /** Callback to be executed when the ArrowItem is clicked or activated by keyboard. */
+  onClick: PropTypes.func,
+
+  /** What type of arrow to display */
+  type: PropTypes.string,
+};
+
+const defaultProps = {
+  a11yText: null,
+  isDisabled: false,
+  onClick: () => {},
+  type: "right",
+};
+
+function ArrowItem(props) {
+  const { a11yText, isDisabled, onClick, type } = props;
+  const Icon = getArrowIcon(`Arrow-${type}`);
+  const I18n = useI18n();
+
+  return (
+    <sc.ArrowItem aria-label={a11yText || I18n.t(`pagination.${type}`)} isDisabled={isDisabled} onClick={onClick}>
+      <Icon isDisabled={isDisabled} size={14} />
```

Comment:
Although icon components support `size` prop, but using styled-component to set the `font-size` should still be the first option
example: https://github.com/acl-services/paprika/blob/master/packages/Calendar/src/internal/calendar.styles.js#L9-L12